packages/create-widget.template.minimal-react.v2/templates/src/main.css

```diff
@@ -1,4 +1,3 @@
 html, body {
-    height: 100%;
```

Comment:
I removed this so that both templates start with taking up as much space as their content which will work better with the default layout of auto in workshop